tests/unit/test_tf_dataloader.py

```diff
@@ -28,8 +26,7 @@
 @pytest.mark.parametrize("gpu_memory_frac", [0.01, 0.1])
 @pytest.mark.parametrize("engine", ["parquet"])
 @pytest.mark.parametrize("batch_size", [1, 10, 100])
-def test_tf_gpu_dl(tmpdir, datasets, batch_size, gpu_memory_frac, engine):
-    paths = glob.glob(str(datasets[engine]) + "/*." + engine.split("-")[0])
+def test_tf_gpu_dl(tmpdir, client, paths, dataset, batch_size, gpu_memory_frac, engine):
```

Comment:
I just installed tensorflow and this test fails for me.  Your suspicion (mentioned offline) that CI is skipping this is likely true.  `KerasSequenceDataset` is expecting the user to pass in raw paths so it can create its own `GPUDatasetIterator`.   Therefore, it is fine to use the dask-based workflow to update the statistics, but we will need to update the dataloader to also accept the same `dataset` object as the workflow (and use `to_iter` to iterate through the dataset)

Reply:
I agree that we're going to need to make those updates you mentioned - but the unittests *should* work with these changes. I've got TF installed on my system, and it works for me:

```
tests/unit/test_tf_dataloader.py ......                                                                          [100%]

=================================================== warnings summary ===================================================
tests/unit/test_tf_dataloader.py: 19 tests with warnings
  /home/ben/code/nvtabular/nvtabular/io.py:552: UserWarning: get_memory_info is not supported. Using total device memory from NVML.
    warnings.warn("get_memory_info is not supported. Using total device memory from NVML.")

-- Docs: https://docs.pytest.org/en/latest/warnings.html
=========================================== 6 passed, 19 warnings in 12.20s ============================================
```

Were you running the TF tests on this branch? What was the error you saw?

Reply:
Ah - my mistake.  You are still passing in `paths` here - makes sense.